2017/11.md

```diff
@@ -86,34 +86,34 @@ Supporting materials includes slides, a link to the proposal repository, a link
         1. [`String.prototype.codePoints`](https://github.com/RReverser/string-prototype-codepoints) for Stage 1 (Ingvar Stepanyan, Mathias Bynens)
         1. [`String.prototype.replaceAll`](https://github.com/psmarshall/string-replace-all-proposal) for Stage 1 (Mathias Bynens)
         1. [Throw expressions](https://github.com/tc39/proposal-throw-expressions#readme) request for reviewers for Stage 3 (Brian Terlson, Ron Buckton).
         1. Additional Intl options, for addition by [needs-consensus PR](https://github.com/tc39/ecma402/pull/175) (Daniel Ehrenberg) ([some](https://github.com/tc39/ecma402/issues/163) [possible](https://github.com/tc39/ecma402/issues/186) [further](https://github.com/tc39/ecma402/issues/95) [options](https://github.com/tc39/ecma402/issues/164))
+        1. [BigInt](https://github.com/tc39/proposal-bigint) status update (Daniel Ehrenberg) ([slides](https://docs.google.com/presentation/d/1u2xXRokUBPMjBsTL_ZCDghA16cH5FI0m1kLOczm1oMw/edit#slide=id.p))
+        1. [Intl.ListFormat](https://github.com/tc39-transfer/proposal-intl-list-format) find Stage 3 reviewers (Daniel Ehrenberg)
     1. 30 Minute Items
         1. [RegExp Unicode property escapes](https://github.com/tc39/proposal-regexp-unicode-property-escapes) for Stage 4 (Mathias Bynens)
         1. [Make ECMAScript a syntactic superset of JSON](https://github.com/gibson042/ecma262-proposal-json-superset) for Stage 2 (by Richard Gibson. Championed by Mark S. Miller & Mathias Bynens)
-        1. [Intl.Locale](https://github.com/tc39/proposal-intl-locale) for Stage 2 (Daniel Ehrenberg)
-        1. Clarify/redefine Stage 4 requirements (Daniel Ehrenberg) ([PR](https://github.com/tc39/process-document/pull/15))
-        1. [Pipeline Operator](https://github.com/tc39/proposal-pipeline-operator) for Stage 2 (Daniel Ehrenberg) ([slides](https://docs.google.com/presentation/d/112oOoEQi1v-uP1jSVWVHCAK7oPA15VX_UP_yarzMeLI/edit#slide=id.p))
-        1. [Allow await in non-arrow async function parameter defaults](https://github.com/tc39/ecma262/issues/917) (Daniel Ehrenberg)
         1. Revisiting parameter initializers and sloppy eval (Adam Klein) ([gist](https://gist.github.com/ajklein/b947351835cc77ad0040db9a55813f51))
         1. Discuss module order instantiation/evaluation guarantees (Bradley Farias) ([slides](https://docs.google.com/presentation/d/1RXvvScD8ce2FyLY2aYhbas83WCiBqzIOqdMt4OpkCJM/view))
         1. Repair loss of Proxy transparency, for Stage 1 (Mark S. Miller, Caridy Patino, Keith Miller, Tom Van Cutsem) ([issue thread](https://github.com/tvcutsem/es-lab/issues/21))
         1. Array`[@@Species]`, Array Index Accessors and Security (Natalie Silvanovich)
+        1. [Intl.Locale](https://github.com/tc39/proposal-intl-locale) for Stage 2 (Daniel Ehrenberg)
+        1. Clarify/redefine Stage 4 requirements (Daniel Ehrenberg) ([PR](https://github.com/tc39/process-document/pull/15))
```

Comment:
I think this topic should be discussed first, as other agenda items depend on its resolution.

Reply:
Would first among 30-minute items be enough, or does it need to be even further forward?

Reply:
Even further IMHO. It should be one of the first items of the entire meeting.

Reply:
@ljharb How would you feel about that?

Reply:
Seems appropriate to me; having it jump the queue seems like the sort of thing @RexJaeschke should/would decide.

Reply:
The stage 4 requirements are of particular interest to me. Would it be possible to move the discussion to after noon on the first day? Due to traveling constraints, I will, unfortunately, be late the first day.

Reply:
@ljharb Currently, the chair group works as a team; it's not one of us making all the decisions. Initially, Rex was proposing that the three of us meet ahead of time to decide on agenda priorities and ordering. However, feedback from some on the committee (including @ljharb) was that we should stick with the rules-based approach. So, as such, we aren't organizing those pre-meetings.

Given that, I think we should discuss here on GitHub whether it's OK to do such a reordering. If you need chair group signoff, I can provide that; I wouldn't expect others to be opposed to this either.

Reply:
Good point :-)

I think it's totally OK; in past meetings we frequently reorder things as needed for scheduling constraints, or because a topic depends on the result of another topic.

This one affects anything in stage 3 or 4, so it both makes sense to prioritize it, and also I can't imagine anyone would object to discussing it first.